index.js

```diff
@@ -13,18 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const fs = require('fs');
+const path = require('path');
 const {Playwright} = require('./lib/server/playwright.js');
+const {localDownloadOptions} = require('./download-browser.js');
 
 const playwright = new Playwright({
   browsers: ['webkit', 'chromium', 'firefox'],
 });
 
-try {
-  const downloadedBrowsers = require('./.downloaded-browsers.json');
-  playwright.chromium._executablePath = downloadedBrowsers.crExecutablePath;
-  playwright.firefox._executablePath = downloadedBrowsers.ffExecutablePath;
-  playwright.webkit._executablePath = downloadedBrowsers.wkExecutablePath;
-} catch (e) {
+if (fs.existsSync(path.join(__dirname, '.local-browsers'))) {
```

Comment:
Why this? Do we still expect playwright-core to work without external setup?

Reply:
this `index.js` is also used when playwright is installed from github (e.g. in case of our local development)